tensorboard/uploader/proto/write_service.proto

```diff
@@ -287,15 +289,47 @@ message GetOrCreateBlobSequenceResponse {
   string blob_sequence_id = 1;
 }
 
+message GetBlobMetadataRequest {
+  // The ID of the BlobSequence of which this blob is a member.
+  string blob_sequence_id = 1;
+  // The position of this Blob within the BlobSequence
+  int64 index = 2;
+}
+
+message GetBlobMetadataResponse {
+  enum BlobState {
```

Comment:
Only noticed now, but let's just make this a top-level symbol (not inside any single response), which is recommended by the proto style guide anyway: https://developers.google.com/protocol-buffers/docs/style#enums

That way we can reuse it across messages easily if we need to (though I'm not sure we do right now).

Even though it's small, might be worth putting it in a separate resource-message .proto file (`blob.proto`) so that it can be shared across services as well, e.g. hypothetically if we want to have the export service export the blob state, since as we discovered with Experiment it's a bit of a pain to relocate later.

Reply:
Haha I had just reverted the top-level enum thing :)  Happy to reinstate that, especially in another file.